.github/ISSUE_TEMPLATE/new-feature.yml

```diff
@@ -0,0 +1,35 @@
+name: New Content
+description: For new features or content suggestions
```

Comment:
For consistency with the other template descriptions, this should start with a verb.